src/spikeinterface/extractors/neoextractors/biocam.py

```diff
@@ -57,11 +59,17 @@ def __init__(
         self.set_property("row", self.get_property("contact_vector")["row"])
         self.set_property("col", self.get_property("contact_vector")["col"])
 
-        self._kwargs.update({"file_path": str(file_path), "mea_pitch": mea_pitch, "electrode_width": electrode_width})
+        self._kwargs.update(
+            {
+                "file_path": str(Path(file_path).absolute()),
+                "mea_pitch": mea_pitch,
+                "electrode_width": electrode_width,
+            }
+        )
 
     @classmethod
     def map_to_neo_kwargs(cls, file_path):
-        neo_kwargs = {"filename": str(file_path)}
+        neo_kwargs = {"filename": str(Path(file_path).absolute())}
```

Comment:
I do not think that all the `map_to_neo_kwargs` were needed.
Only the `self._kwargs` was the problem.
This doesn't hurt but this make the read a bit strange

Reply:
right, it does simplify things

Reply:
done